test/expression/micro/parser_test.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:angular_ast/src/ast.dart';
+import 'package:angular_ast/src/expression/micro/ast.dart';
+import 'package:angular_ast/src/expression/micro/parser.dart';
+import 'package:test/test.dart';
+
+void main() {
+  NgMicroAst parse(String directive, String expression) {
+    return const NgMicroParser().parse(
+      directive,
+      expression,
+      sourceUrl: '/test/expression/micro/parser_test.dart#inline',
+    );
+  }
+
+  test('should parse a simple let', () {
+    expect(
+      parse('ngThing', 'let foo'),
+      new NgMicroAst(
+        assignments: [
+          new ReferenceAst('foo'),
+        ],
+        properties: [],
+      ),
+    );
+  });
+
+  test('should parse a let assignment', () {
+    expect(
+      parse('ngThing', 'let foo = bar;let baz'),
```

Comment:
By the way, I managed to get this test case working. Don't think we need to support it as far as I'm aware, but its technically currently valid due to the whole `*x="y"` being equivalent to `template="x y"` thing.

template:
```
<div *templateComp="'expression'; plus: 'second expression'; let templateComp">
    &lt;div *templateComp="{{templateComp.templateComp}}; plus: {{templateComp.templateCompPlus}}; let templateComp"&gt; worked
</div>
```

output:
```
<div *templateComp="expression; plus: second expression; let templateComp"> worked
```

(technically it showed that twice for some bug but that's unimportant)

dart:
```
@Directive(selector: '[templateComp]', exportAs: 'templateComp')
class TemplateComponent {

  String _templateComp;
  String _templateCompPlus;
  TemplateRef _templateCompTemplate;
  @Input() set templateComp(String x) { _templateComp = x; rerender(); }
  @Input() set templateCompPlus(String x) { _templateCompPlus = x; rerender(); }
  @Input() set templateCompTemplate(TemplateRef x) { _templateCompTemplate = x; rerender(); }
  String get templateComp => _templateComp;
  String get templateCompPlus => _templateCompPlus;

  ViewContainerRef _viewContainer;

  TemplateComponent(this._viewContainer, this._templateCompTemplate);

  void rerender() {
    var view = this._viewContainer.createEmbeddedView(_templateCompTemplate);
    view.setLocal(r'$implicit', this);
    _viewContainer.insert(view);
  }

}
```